From 360235fcc0f5c821ad9d99ccbe5b0b465d9c85fa Mon Sep 17 00:00:00 2001
From: Chen Minqiang <ptpt52@gmail.com>
Date: Fri, 2 Jul 2021 17:59:05 +0800
Subject: [PATCH] miniupnpd: add force_forwarding option support

This option replace the 'ext_ip_reserved_ignore', and replace the
patch with new one

as addr_is_reserved() behavior should not be changed, so drop the
ext_ip_reserved_ignore

use a force_forwarding option to make the port forwarding force to
work when the router is behind NAT

by default force_forwarding is set to 0(disabled), where change
nothing with the original behavior, and users can turn it on
according to their wishes

Signed-off-by: Chen Minqiang <ptpt52@gmail.com>
---
 net/miniupnpd/files/upnpd.config              |   1 +
 1 file changed, 1 insertion(+)

diff --git a/net/miniupnpd/files/upnpd.config b/net/miniupnpd/files/upnpd.config
index bd7c3ec..b03d60a 100644
--- a/net/miniupnpd/files/upnpd.config
+++ b/net/miniupnpd/files/upnpd.config
@@ -2,6 +2,7 @@ config upnpd config
 	option enabled		0
 	option enable_natpmp	1
 	option enable_upnp	1
+	option ext_allow_private_ipv4 1
 	option secure_mode	1
 	option log_output	0
 	option download		1024
